public/src/components/Main.tsx

```diff
@@ -550,6 +568,7 @@ export class Main extends React.Component<MainProps, MainState> {
                     stockBriefInfoMap={this.state.stockBriefInfoMap}
                     stockPricesMap={this.getStockPrices(this.state.stockDetailsMap)}
                     stocksOwnedMap={this.state.stocksOwnedMap}
+                    reservedStocksWorth={this.calculateUserReservedStocksWorth(this.state.stockDetailsMap, this.state.stocksReservedMap)}
```

Comment:
Make `reservedStocksworth` a state variable so that the value is not calculated again and again.